packages/experiments/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "sourceMap": true,
     "experimentalDecorators": true,
     "importHelpers": true,
+    "noUnusedLocals": true,
```

Comment:
We should create a base tsconfig and have all projects extend that base config

Reply:
We have that...

Reply:
Oops nevermind thats tslint